test/test_convolutional_layer.h

```diff
@@ -13,11 +13,11 @@
 namespace tiny_dnn {
 
 TEST(convolutional, setup_internal) {
-  convolutional_layer<sigmoid> l(5, 5, 3, 1, 2);
+  convolutional_layer l(5, 5, 3, 1, 2);
 
   EXPECT_EQ(l.parallelize(), true);              // if layer can be parallelized
   EXPECT_EQ(l.in_channels(), serial_size_t(3));  // num of input tensors
-  EXPECT_EQ(l.out_channels(), serial_size_t(2));    // num of output tensors
+  EXPECT_EQ(l.out_channels(), serial_size_t(1));    // num of output tensors
```

Comment:
`out_channels` returns the number of output vectors of tensors. decoupled layers don't need `vector_type::aux` anymore, so it reduced.

Although I think `out_channels` name is misleading. @edgarriba @Randl thoughts ?